common.blocks/workspace/workspace.js

```diff
@@ -17,18 +17,24 @@ modules.define('workspace', ['i-bem__dom', 'jquery', 'info-modal'],
                     InfoModal.show(response);
                 });
         },
-        _getTranslation: () => {
-            $.get('/translation' + location.search)
+
+        _getTranslation: function() {
+            const data = this._editor.provideData({ onlyTranslated: false });
+            InfoModal.show('Loading');
+            $.post('/translate', { data: JSON.stringify(data) })
                 .then((response) => {
-                    console.log(response);
+                    InfoModal.hide();
+                    BEMDOM.replace(this._editor.domElem, response);
```

Comment:
I think variable _editor need update for example: this._editor = this.findBlockInside('editor');